src/lib/plot/boxwhisker.typ

```diff
@@ -0,0 +1,90 @@
+#import "../../draw.typ"
+
+#let add-boxwhisker(
+    data,
+    axes: ("x", "y"),
+    style: (:),
+    box-width: 0.75,
+    whisker-width: 0.5,
+    mark: "*",
+    mark-size: 0.2
+) = {
+
+    if type(data) == array {
+        for it in data{
+            add-boxwhisker(
+                it,
+                axes:axes,
+                style: style,
+                box-width: box-width,
+                whisker-width: whisker-width,
+                mark: mark,
+                mark-size: mark-size
+            )
+        }
+        return
+    }
+
+    assert( "x" in data, message: "Specify the x value at which to display the box and whisker")
+    assert( "min" in data, message: "Specify the q1, the minimum excluding outliers")
+    assert( "q1" in data, message: "Specify the q1, the lower quartile")
+    assert( "q2" in data, message: "Specify the q2, the median")
+    assert( "q3" in data, message: "Specify the q3, the upper quartile")
+    assert( "max" in data, message: "Specify the q1, the minimum excluding outliers")
+
+    // Calculate y-domain
+
+    let max-value = calc.max(
```

Comment:
(same for `min-value): The `0` is not correct here. Use `util.max` and `util.min` from `/src/util.typ`, as they ignore `none` values.
```
let max-value = util.max(data.max, data.at("outliers"))
```